libs/components/src/next/properties/stories/vl-properties-stories-doc.mdx

```diff
@@ -0,0 +1,108 @@
+import {ArgTypes, Canvas} from '@storybook/addon-docs';
+
+import * as VlPropertiesStories from './vl-properties.stories';
+
+# Properties - next
+
+Gebruik `properties-next` om properties te tonen op een pagina. Je kan de properties specifiëren als inner-html of
+als json-structuur meegeven aan het `prop` attribuut.
+
+Deze component is de nieuwste variant die de (nog als element geschreven)
+[vl-properties](/docs/elements-properties--documentatie) vervangt.
+
+## Voorbeeld
+
+```js
+import { VlPropertiesComponent } from '@domg-wc/components/next/properties';
+```
+
+```html
+<vl-properties-next></vl-properties-next>
+```
+
+<Canvas of={VlPropertiesStories.PropertiesDefault}/>
+
+
+## Configuratie
+
+<ArgTypes of={VlPropertiesStories.PropertiesDefault}/>
+
+
+### Props
+
+De props zijn als volgt gespecifiëerd:
+```ts
+export type Props = Column[];
+
+export interface Column {
+    class?: string; // column / column--full-width / collapsed
```

Comment:
'column' | 'column--full-width' | 'collapsed'

Reply:
Zoals besproken met Kris, is `column column--full-width` ook een optie, daarom blijft `string` hier beter.